scripts/globus/setup_collection_directory.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -euf -o pipefail
+
+SCRIPT=$(realpath "$0")
```

Comment:
I would avoid using "$0" and would recommend instead using ${BASH_SOURCE}.

Reply:
Behavior in Sourced Scripts:
If a script is sourced (using . script.sh or source script.sh), $0 refers to the name of the parent shell, not the script being sourced.

Reply:
Updated